modules/contact/doc/content/modules/contact/index.md

```diff
@@ -270,3 +270,38 @@ The recommended PETSc options for use with `NodeFaceConstraint` based contact ar
 ## Objects, Actions, and Syntax
 
 !syntax complete groups=ContactApp level=3
+
+## Scaling effects
+
+The effects of scaling on non-linear solve convergence are shown below for
+different kinds of contact formulations. The results are based on the input files for
+[RANFS](/ranfs-and-scaling/bouncing-block-ranfs.i),
+[kinematic](/bouncing-block-kinematic.i), and
+[tangential penalty](/bouncing-block-tan-pen.i),
+running with two processes.
+
+| Scheme | Cumulative nonlinear iterations | Cumulative linear iterations |  Condition number | Scaling Factor |
+| - | - | - | - | - |
+| SMP PJFNK RANFS no scaling AMG | 61 | 310 | 6e3 | 1 |
```

Comment:
Shouldn't you just say 'N/A' for the scaling factor for the RANFS cases?

Reply:
No because `Scaling Factor` refers to the number used for variable scaling. I'll do this change: `Scaling Factor` -> `Variable Scaling Factor`

Reply:
OK, yeah -- when I first saw this, I thought you were talking about a penalty factor.